libs/common/src/assets/i18n/en.json

```diff
@@ -687,6 +687,8 @@
 	"stash.valid_emails_label": "Emails",
 	"stash.view_edit_service_access": "View / Edit existing Service Access",
 	"stash.widgets": "Widgets",
+	"stash.widgets.answers": "Answer generation",
+	"stash.widgets.answers.help": "Generate answer to user questions and allow to chat with the system",
```

Comment:
`Generate answer to user questions` seems weird, no ? Did you mean `Generate answer from user questions` ?